### DISCUSSION TITLE
`rbenv install 3.2.3` fails on M3 (macOS 14.4.1)

### DISCUSSION THREAD

**Discussion by yhqpanda** [rbenv/ruby-build#2364] [Build failures]
### Machine details:
- Apple M3 Pro
- macOS 14.4.1

### Error encountered:
`BUILD FAILED (macOS ... )`

### Steps taken to reproduce error:
1. Install `homebrew`
2. `brew install rbenv ruby-build` (https://github.com/rbenv/rbenv?tab=readme-ov-file#installation)
3. `rbenv install 3.2.3`

### Steps taken to resolve error:
Follow all steps in this guide https://medium.com/@nelliemckesson/install-ruby-with-rvm-on-an-m3-pro-mac-784718bdb72a, except final step to do `rvm install 3.2.3 --with-openssl-dir=/usr/local/opt/openssl@1.1`

Feel free to share if there are other solutions!

**Comment by mislav** [comment]
Hi, the error message you pasted `BUILD FAILED (macOS)` isn't nearly enough for us to understand what might have gone wrong. You would need to look inside the full build log (written to a file) and search for messages there that mention "error".

Also, you seem to have been trying `rvm install 3.2.3`. Note that rbenv and RVM are competing version managers for Ruby and that you should not be using both at the same time. You should pick one and uninstall the other, as they might be conflicting in some areas, leading to hard-to-debug problems.
